integrations/amazon_bedrock/src/haystack_integrations/components/generators/amazon_bedrock/chat/chat_generator.py

```diff
@@ -295,10 +293,11 @@ def from_dict(cls, data: Dict[str, Any]) -> "AmazonBedrockChatGenerator":
     def _prepare_request_params(
         self,
         messages: List[ChatMessage],
-        streaming_callback: Optional[Callable[[StreamingChunk], None]] = None,
+        streaming_callback: Optional[StreamingCallbackT] = None,
         generation_kwargs: Optional[Dict[str, Any]] = None,
         tools: Optional[List[Tool]] = None,
-    ) -> Tuple[Dict[str, Any], Optional[Callable[[StreamingChunk], None]]]:
+        requires_async: bool = False,
```

Comment:
We should add the new `requires_async` parameter to the docstring to keep it up to date.

Reply:
@julian-risch thanks! I've just updated it